tests/MartinGeorgiev/Doctrine/DBAL/Types/BaseTypeTest.php

```diff
@@ -26,7 +26,7 @@ protected function setUp(): void
         $this->platform = $this->createMock(AbstractPlatform::class);
 
         $this->fixture = $this->getMockBuilder(BaseType::class)
-            ->setMethods(null)
+            ->setMethods()
```

Comment:
I'm not sure why this change is needed.

Reply:
PHPUnit type hints this to `array` (not `?array`) and passing `null` is throwing a type error

Reply:
In true Sunday morning style, the original commit of mine was also wrong, it needs an empty array passed.